src/BlockchainWriter/Controller.ts

```diff
@@ -15,6 +15,7 @@ import { ExchangeConfiguration } from './ExchangeConfiguration'
 export interface ControllerConfiguration {
   readonly poetNetwork: string
   readonly poetVersion: number
+  readonly maxBlockHeightDelta: number
```

Comment:
I like `maxBlockHeightDelta` but I'm thinking whitout context it could refer to anything. Maybe `maxTransactionAgeInBlocks`?

Reply:
ok I'm not sure that's too much clearer, but I'll change